backend/flask_main.py

```diff
@@ -124,3 +118,5 @@ def current_users():
         }
         redis_client.publish('flask-IPC', json.dumps(ipc_message))
         return user_dict
+    else:
+        return '', HTTP_STATUS_CODE.HTTP_404_NOT_FOUND.value
```

Comment:
Log - sure.  I've got my best Jacob on that [SCRUM-56].
Otherwise really this should be an HTTP 405, method not allowed.  But even more actually, this will never get called because the decorator for the route already disallows anything but GET and POST - some of this was provisional, danny learning flask that does need to be cleaned up.  Not much use sending a body in 404's or 405's IMO but not entirely not debateable